pkg/controller/topology_test.go

```diff
@@ -876,6 +880,23 @@ func TestTopologyAggregation(t *testing.T) {
 				{Segments: map[string]string{"com.example.csi/zone": "zone2"}},
 			},
 		},
+		"selected node; different values across cluster with strict topology": {
```

Comment:
Can you add the case with no selected node and strict topology? Actually, all previous testcases should be duplicated and compared with strict topology, to ensure that they work as expected.

Reply:
I made changes to tests, such that now every topology test runs with and without strict topology.